classes/SuperDirt.sc

```diff
@@ -284,14 +284,22 @@ SuperDirt {
 			}
 		};
 
-
 		netResponders.add(
+			OSCFunc({ |msg, time, tidalAddr|
+				var name = msg[1].asSymbol;
+				var synth = SynthDescLib.global.at(name);
+				var controls = synth.controls;
+				var controlNames = msg[1];
+				controls.do{arg control; controlNames = controlNames ++ " " ++ control.name};
```

Comment:
`controls.do{ |control| ` is nicer, if you like.

Reply:
done thanks!